sdk/servicebus/Microsoft.Azure.ServiceBus/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Release History
+## 5.0.0
```

Comment:
You will need to add the date of release after the version, similar to below.

Reply:
Date is optional as its not easy to pick a specific date.

---
In reply to: [486693199](https://github.com/Azure/azure-sdk-for-net/pull/15073#discussion_r486693199) [](ancestors = 486693199)

Reply:
The CI will fail if there is no date:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=532847&view=logs&jobId=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=80694584-9a00-5514-2ba2-f6dabf815050